main/src/main/java/cgeo/geocaching/CachePopupFragment.java

```diff
@@ -199,6 +199,33 @@ public void onConfigurationChanged(@NonNull final Configuration newConfig) {
         init();
     }
 
+
+    private void doStoreCacheOnLists(final Set<Integer> listIds) {
```

Comment:
I moved the method outside the Listener-class to have access from the move-listener as well